hack/create_mock_bundles.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
```

Comment:
`set -eou pipefail`

will exit if there is an error.

Reply:
thanks for the catch, adding